project_one.css

```diff
@@ -888,6 +1001,96 @@
     border: 1px solid #cfd8dc;
     box-shadow: 5px 5px #8490a7;
   }
+  #popup {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: rgba(236, 236, 236, 0.9);
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 9;
+  }
+
+  .pop-card {
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    font-family: "Poppins", sans-serif;
+    background: #ffffff;
+    transition: 0.5s;
+    width: 80%;
+  }
```

Comment:
- [ ] It would be nicer to margins to the top and the bottom of the pop-up card so your design matches the Figma design:
- Your design:
<img width="947" alt="image" src="https://user-images.githubusercontent.com/77190203/162045839-91edeb91-2dc8-42db-82c5-8108740ae595.png">

- Figma design:
<img width="296" alt="image" src="https://user-images.githubusercontent.com/77190203/162045774-45a71696-a176-4b3e-9da4-9fc6ac35daad.png">

Reply:
Thanks, sir for highlighting my mistakes. I'll improve it